index.html

```diff
@@ -39,64 +54,113 @@ <h2>I am a Software Developer</h2>
   <section class="work">
```

Comment:
As per the requirements of the project, you should use both `flexbox` and `grid` to make the layout, however, you have only used `flexbox`. I would recommend, using `grid` for outer main sections and `flexbox` to layout the inner components, in this case, the individual projects.